app/views/users/stores/_store.html.erb

```diff
@@ -0,0 +1,5 @@
+<tr id="store-<%= store.id %>" class="store">
+  <th scope="row">Insert Image Here</th>
```

Comment:
Is this an image for the store? 
It's unclear to me what this is for.

Reply:
pretty much, i'm assuming that when we have to create a stores#index and show page, we will need images, I'll take that line out for now